src/coreclr/vm/pgo.h

```diff
@@ -3,6 +3,428 @@
 
 #ifndef PGO_H
 #define PGO_H
+#include "typehashingalgorithms.h"
+#include "shash.h"
+
+
+enum class PgoInstrumentationKind
```

Comment:
My original thought was that this one would only have the minimal number of enum values describing marshalling so that it was clear that the JIT defined the meanings of stuff, but honestly, that's not a major benefit. I'll merge the two.